src/pages/questionAnswerViewer/QuestionAnswerViewer.jsx

```diff
@@ -106,7 +109,32 @@ export default function QuestionAnswerViewer() {
     }
   }
 
-  useEffect(() => { fetchAnswers(); }, [user, question_id, answer_id]);
+  function handleAnswerDeleted(response) {
+    if (response.status === 'error') {
+      displayAlert('error', response.message);
+    } else {
+      displayAlert('success', 'Deleted Answer');
+      history.push(`/question/${question_id}`);
+    }
+  }
+
+  useEffect(() => {
+    fetchAnswers();
+  }, [user, question_id, answer_id]);
+
+  async function getNewAnswer() {
+    const response = await API.queryDispatcher.getAnswer(question_id, user.id_token);
+    if (response.status === 'error') {
+      pageStatus.setFailure(response.message);
+      return;
+    }
+    displayAlert(
+      'success',
+      <Button onClick={() => history.replace(`/question/${question_id}/answer/${response.id}`)}>
+        A new answer is ready!
```

Comment:
It's not fully clear that this is a button. Maybe we should have two lines, first line says "A new answer is ready!" and the second line is a button that says "Go" and takes you to the answer.